src/bp/ui-studio/src/web/views/Content/index.tsx

```diff
@@ -64,6 +79,46 @@ class ContentView extends Component<Props, State> {
   }
 
   currentContentType() {
+    this.props.contentItems.forEach(async (element: ContentElementUsage) => {
+      element.usage = []
+      Object.values(this.props.flows.flowsByName).forEach((flow: FlowView) => {
+        const name = getFlowLabel(flow.name)
+        flow.nodes.forEach((node: NodeView) => {
+          const usage: ContentUsage = {
+            type: 'Flow',
+            name,
+            node: node.name,
+            count: 0
+          }
+          node.onEnter &&
+            node.onEnter.forEach((v: string | ActionBuilderProps) => {
+              if (typeof v === 'string' && v.startsWith('say #!' + element.id)) {
+                if (!usage.count) {
+                  element.usage.push(usage)
+                }
+                usage.count++
+              }
+            })
+          node.onReceive &&
+            node.onReceive.forEach((v: string | ActionBuilderProps) => {
+              if (typeof v === 'string' && v.startsWith('say #!' + element.id)) {
+                if (!usage.count) {
+                  element.usage.push(usage)
+                }
+                usage.count++
+              }
+            })
```

Comment:
Those two blocs are duplicated, you could put the code for forEach.,... in a method, then you could call it twice, once for onEnter, then onReceive